pkg/scheduler/framework/runtime/framework.go

```diff
@@ -73,6 +74,9 @@ type frameworkImpl struct {
 	postBindPlugins      []framework.PostBindPlugin
 	permitPlugins        []framework.PermitPlugin
 
+	instrumentedPreFilterPlugins []instrumentedPreFilterPlugin
+	instrumentedFilterPlugins    []instrumentedFilterPlugin
```

Comment:
let's get rid of the other lists.

```suggestion
	preFilterPlugins []instrumentedPreFilterPlugin
	filterPlugins    []instrumentedFilterPlugin
```

Reply:
Done.